pipeline/inputs/statsd.md

```diff
@@ -16,6 +16,12 @@ The plugin supports the following configuration parameters:
 | Listen | Listener network interface. | 0.0.0.0 |
 | Port | UDP port where listening for connections | 8125 |
 | Threaded | Indicates whether to run this input in its own [thread](../../administration/multithreading.md#inputs). | `false` |
+| Metrics | Ingest as metrics type of events. | `off` |
+
+Note: When enabling `Metrics On`, the DogStatsD protocol of metrics can be handled.
+Including key-value format of tags:
+`<METRIC_NAME>:<VALUE>|<TYPE>|@<SAMPLE_RATE>|#<TAG_KEY_1>:<TAG_VALUE_1>`
+Events and ServiceChecks format are not supported yet with `Metrics On`.
```

Comment:
What are these? Can we link to their definition?

Reply:
I added links for those of definitions: https://github.com/fluent/fluent-bit-docs/pull/1542/commits/97c73c67311408ddaaf97a1e59e5524e199dab1b